hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/clean/CleanPlanActionExecutor.java

```diff
@@ -152,9 +155,19 @@ protected Option<HoodieCleanerPlan> requestClean(String startCleanTime) {
         LOG.error("Got exception when saving cleaner requested file", e);
         throw new HoodieIOException(e.getMessage(), e);
       }
-      return Option.of(cleanerPlan);
+      option = Option.of(cleanerPlan);
     }
-    return Option.empty();
+
+    if (config.isTableManagerSupportAction(ActionType.clean)) {
+      createCleanToService();
```

Comment:
rename: delegateCleanExecutionToTableManager() ? In general, lets strive for consistent naming of similar concepts across the PR. e.g pick of of the following - `TableManagerService`, `TableManager`, `Service` (probably not ideal since we dont know what "service" is)

Reply:
Understand, I will uniformly name the `Service` to `TableManagerService`.